setup.py

```diff
@@ -13,7 +13,7 @@
     description='Sphinx extension to build a 404 page with absolute URLs',
     url='https://github.com/readthedocs/sphinx-notfound-page',
     license='MIT',
-    packages=setuptools.find_packages(),
+    packages=setuptools.find_packages(exclude=["common*"]),
```

Comment:
Is it possible to use the `MANIFEST.in` for this? I'd like to have all the exclude/include in the same place if possible.

Reply:
`MANIFEST.in` are for source distributions, not wheels 😅 https://packaging.python.org/guides/using-manifest-in/